File: app/src/main/java/com/example/chatgptapp/network/RetrofitInstance.java
package com.example.chatgptapp.network

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitInstance {
        val api: ChatGptService by lazy {
        Retrofit.Builder()
        .baseUrl("https://url-da-api") // Substitua pela URL base da API do ChatGPT
        .addConverterFactory(GsonConverterFactory.create())
        .build()
        .create(ChatGptService::class.java)
        }
        }

File: app/src/main/java/com/example/chatgptapp/network/RetrofitInstance.kt
package com.example.chatgptapp.network

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitInstance {
    private const val BASE_URL = "https://api.openai.com/v1/" // URL base da API do ChatGPT

    val api: ChatGptService by lazy {
        Retrofit.Builder()
            .baseUrl(BASE_URL)
            .addConverterFactory(GsonConverterFactory.create())
            .build()
            .create(ChatGptService::class.java)
    }
}
